docs/content/latest/explore/observability/prometheus-integration/linux.md

```diff
@@ -57,24 +57,37 @@ Prometheus is installed on your local machine. If you have not done so already,
 
 ## 1. Create universe
 
-If you have a previously running local universe, destroy it using the following.
+Start a new local YugabyteDB cluster - this will create a three-node universe with a replication factor of `3`.
 
 ```sh
-$ ./bin/yb-ctl destroy
+$ ./bin/yugabyted start \
+                  --base_dir=node-1 \
+                  --listen=127.0.0.1 \
+                  --tserver_flags="start_redis_proxy=true"
 ```
 
-Start a new local YugabyteDB cluster - this will create a three-node universe with a replication factor of `3`.
-
 ```sh
-$ ./bin/yb-ctl create --rf 3
+$ ./bin/yugabyted start \
+                  --base_dir=node-2 \
+                  --listen=127.0.0.2 \
+                  --join=127.0.0.1 \
+                  --tserver_flags="start_redis_proxy=true"
+```
+
+```
+$ ./bin/yugabyted start \
+                  --base_dir=node-3 \
+                  --listen=127.0.0.3 \
+                  --join=127.0.0.1 \
+                  --tserver_flags="start_redis_proxy=true"
 ```
 
 ## 2. Run the YugabyteDB workload generator
 
 Download the [YugabyteDB workload generator](https://github.com/yugabyte/yb-sample-apps) JAR file (`yb-sample-apps.jar`) by running the following command.
 
 ```sh
-$ wget https://github.com/yugabyte/yb-sample-apps/releases/download/1.3.1/yb-sample-apps.jar?raw=true -O yb-sample-apps.jar
+$ wget https://github.com/yugabyte/yb-sample-apps/releases/download/1.3.2/yb-sample-apps.jar?raw=true -O yb-sample-apps.jar
```

Comment:
same as the other one: can we use a "latest" link that won't have to change?

Reply:
Resolution: no, we can't, because some later versions of the sample apps are broken.